Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -189,7 +190,7 @@ void JitArm64::fp_arith(UGeckoInstruction inst)
     break;
   }
 
-  std::vector<FixupBranch> nan_fixups;
+  Common::SmallVector<FixupBranch, 3> nan_fixups;
```

Comment:
I think this needs to be 4? 

Assuming an instruction for which negate_result is true:

- op5 (inst.SUBOP5) must be 30 or 31 (because that's what makes negate_result true)
- Both use_b and use_c are true when op5 > 25
- can_generate_nan is true because op5 isn't 20 or 21
- Assuming VA, VB, and VC are distinct, inputs will have size 3
- The loop puts 1 element in nan_fixups for each element in inputs, so 3
- Because can_generate_nan and negate_result are true, the next block puts a 4th branch into nand_fixups.

Reply:
Yes. I didn't notice that negate_result caused an extra fixup to be pushed.